### PR TITLE
LVUIDF-66: TinyMCE implementation, 'path' removed.

### PR DESCRIPTION
**Pull request by Fleeck**
https://ssu-jira.softserveinc.com/browse/LVUIDF-66

 - [x] Removed 'path' parameter from Page model.
 - [x] Implemented TinyMCE editor into Page.

To review:
- [ ] @rtriska
- [ ] @forecome 

To merge:
- [ ] @izagor
- [ ] @forecome

### CODE REVIEW

File: app/assets/javascripts/components/pages/page_form.js.jsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ var PageForm = React.createClass({
 
   getInitialState: function() {
     return {page: {}};
+
```
Comment (forecome) [inline app/assets/javascripts/components/pages/page_form.js.jsx]: Please, don't do such useless changes. It makes git history dirty.

File: db/migrate/20150710080653_create_pages.rb [hunk 2]
```diff
@@ -3,8 +3,7 @@ def change
     create_table :pages do |t|
       t.string :title
       t.text :body
-      t.string :path
```
Comment (forecome) [inline db/migrate/20150710080653_create_pages.rb]: It's not good to change existent migration because you will break consistency of DB deployment strategy. Another people who works with you on one project may have troubles with DB migrating.
I know, that's great when you have clean code with clean migrations from the beginning, but you can do such thing only if you work alone on the project and not deployed it anywhere yet (or you are able to reset the DB).
Please, revert this change and create new migration, which removes this column.

Comment (Fleeck) [inline db/migrate/20150710080653_create_pages.rb]: @forecome 
I removed this, dropped the DB, recreated it and migrated once more.

Comment (forecome) [inline db/migrate/20150710080653_create_pages.rb]: On production server you can not do such actions. Please, revert this change and create new migration, which removes the column.

Comment (Fleeck) [inline db/migrate/20150710080653_create_pages.rb]: @forecome reworked into additional branch. Please, review 
https://github.com/dark-side/language-school/pull/43

**Comment by rtriska** [comment]
Close as no longer needed
